runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -421,4 +423,39 @@ private void onTileObject(Tile tile, TileObject oldObject, TileObject newObject)
 			}
 		}
 	}
+
+	@Subscribe
+	public void onMenuEntryAdded(MenuEntryAdded event)
+	{
+		if (config.showShortcutLevel())
+		{
+			MenuEntry[] entries = client.getMenuEntries();
+
+			//Skip anything where there's 0 chance it's a agi shortcut.
+			if (event.getTarget() == null || event.getTarget() == ""){
+				return;
+			}
+
+			Map<TileObject,Obstacle> obstacles = getObstacles();
+			for (Obstacle o : obstacles.values())
+			{
+				//Compare the event ID to nearby obstacle ID's
+				for (int obId: o.getShortcut().getObstacleIds()){
+					if (obId == event.getIdentifier()){
+						AgilityShortcut agilityShortcut = o.getShortcut();
+						for (MenuEntry entry : entries) {
+							//Check it's the non-examine
+							if (entry.getTarget().equalsIgnoreCase(event.getTarget()) && !entry.getOption().equalsIgnoreCase("examine") && !entry.getTarget().toLowerCase().contains("(level-")) {
+								//Append level with a color to the added entry and update it!
+								if (agilityShortcut.getLevel() <= getAgilityLevel()) entry.setTarget(entry.getTarget() + " <col=00ff00>(level-" + agilityShortcut.getLevel() + ")");
```

Comment:
Don't use oneline `if`s.

Reply:
Use `ColorUtil` for the color string building. Also, append the `"(level-" + agilityShortcut.getLevel() + ')'` text outside of this if-else block; it is the same regardless of whether the condition is true.